include/swift/ABI/ObjectFile.h

```diff
@@ -28,10 +28,20 @@ class SwiftObjectFileFormat {
 public:
   virtual ~SwiftObjectFileFormat() {}
   virtual llvm::StringRef getSectionName(ReflectionSectionKind section) = 0;
+  virtual bool sectionContainsReflectionData(llvm::StringRef sectionName) {
```

Comment:
I don't like that the generic implementation is incomplete (missing DATA). Given that this could be more efficiently implemented with a single `sectionName.starts_with("__swift5")` maybe we should make this a pure virtual function?
Given that the "safe" answer here is false, I guess it's okay to not have implementations for ELF and COFF?

Reply:
Okay maybe we can leave this generic implementation for ELF and COFF — it's incomplete but safe, and we can still do the starts_with trick down below.

Reply:
good call, done.